rts/System/Platform/Misc.cpp

```diff
@@ -519,4 +519,20 @@ std::string ExecuteProcess(const std::string& file, std::vector<std::string> arg
 	return execError;
 }
 
+void OpenURL(const std::string& url)
```

Comment:
A regular expression check of the string should verify if it is indeed a valid URL. If we're being strictly OO, the parameter should be a dedicated URL object not an `std::string`